# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/systemd/pretty-print.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

More minimal and pretty status.

--- systemd-258.2/src/core/show-status.c.vanilla	2025-12-11 14:59:12.591701225 +0100
+++ systemd-258.2/src/core/show-status.c	2025-12-11 15:00:20.415804178 +0100
@@ -36,7 +36,7 @@
 }
 
 int status_vprintf(const char *status, ShowStatusFlags flags, const char *format, va_list ap) {
-        static const char status_indent[] = "         "; /* "[" STATUS "] " */
+        static const char status_indent[] = "       "; /* "" STATUS " " */
         static bool prev_ephemeral = false;
         static int dumb = -1;
 
@@ -96,9 +96,8 @@
 
         if (status) {
                 if (!isempty(status)) {
-                        iovec[n++] = IOVEC_MAKE_STRING("[");
                         iovec[n++] = IOVEC_MAKE_STRING(status);
-                        iovec[n++] = IOVEC_MAKE_STRING("] ");
+                        iovec[n++] = IOVEC_MAKE_STRING(" ");
                 } else
                         iovec[n++] = IOVEC_MAKE_STRING(status_indent);
         }
